src/app/children/child-details/child-details.component.ts

```diff
@@ -103,10 +103,11 @@ export class ChildDetailsComponent implements OnInit {
     const params = this.route.snapshot.params;
     const childId = params['id'];
 
-    this.entityMapperService.load(Child, 'child:' + childId)
+    this.entityMapperService.load<Child>(Child, childId)
       .then(child => this.child = child)
       .catch(err => {
-        console.log(err + 'load');
+        console.log(childId);
+        console.log('!!!!!' + err + 'load');
```

Comment:
I assume this will be cleaned up once the real ChildrenDetailsComponent gets ready.
I opened a general issue about error reporting #115 as I also came accross some situations where I wasn't sure where/how to log things.